scripts/example-analysis.jl

```diff
@@ -0,0 +1,29 @@
+# First, read in the output energies and iterations as a DataFrame
+df = read_output("scripts/LJ_111_p08_output_df.csv")
+
+# Calculate the ω factors
+ωi = ωᵢ(df.iter, 640)
+
+# Shift the energies to be positive
+Ei = df.emax .- minimum(df.emax)
+# Specify the temperatures that we are interested in
+Ts = collect(10:1:2000)
+# Define the Boltzmann constant
+kb = 8.617333262e-5 # eV/K
```

Comment:
It doesn't have to be done now, but I'd like us to define this once somewhere in the code and then reference that definition afterward.

Reply:
Good point! I'll move it to a issue.